### DISCUSSION TITLE
CanvasApp import: FAILURE: No file with that name found

### DISCUSSION THREAD

**Discussion by Bjebali** [microsoft/PowerApps-Language-Tooling#693] [Issues]
Solution Importing...
  Microsoft PowerPlatform CLI
  Version: 1.25.2+g1c4a1b8
  Online documentation: https://aka.ms/PowerPlatformCLI
  Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions
  
  Error: CanvasApp import: FAILURE: No file with that name found
      CanvasApp import: FAILURE: No file with that name found

**Comment by marcelbf** [comment]
To source control your canvas app, please use the [Power Platform git integration](https://learn.microsoft.com/power-platform/alm/git-integration/canvas-apps-git-integration).

More information about the [YAML serialization format](https://learn.microsoft.com/power-apps/maker/canvas-apps/power-apps-yaml#power-apps-yaml-schema-versions) is available in the documentation.
